### PR TITLE
Translation done.

### PR DESCRIPTION
**Pull request by jesseke55**
I`ve translated everything right now and I added in the enter forum description added 'HTML Codes kunnen worden gebruikt' Which means you can use HTML codes which may be handy/usefull that I might add in the english translation

### CODE REVIEW

File: styles/language/Dutch/language.php [hunk 1]
```diff
@@ -313,13 +316,13 @@
...
 	
 	// Admin Update page
-	'installation_up_to_date' => 'Your installation is up to date.',
-	'update_check_error' => 'Unable to check for updates. Please try again later.',
-	'new_update_available' => 'A new update is available.',
-	'your_version' => 'Your version:',
-	'new_version' => 'New version:',
+	'installation_up_to_date' => 'Je installatie is up to date',
+	'update_check_error' => 'Helaas konden we niet controleren of er updates zijn. Probeer het later nog eens.',
+	'new_update_available' => 'A new update is available. Er is een update beschikbaar.',
```
Comment (Azoraqua) [inline styles/language/Dutch/language.php]: Verwijder "A new update is available.".

File: styles/language/Dutch/language.php [hunk 2]
```diff
@@ -393,11 +396,11 @@
...
-	'signature_maximum_900' => 'Your signature must be a maximum of 900 characters.',
-	'invalid_date_of_birth' => 'Invalid date of birth.',
-	'location_required' => 'Please enter a location.',
-	'location_minimum_2' => 'Your location must be a minimum of 2 characters.',
-	'location_maximum_128' => 'Your location must be a maximum of 128 characters.',
+	'signature_maximum_900' => 'Je handtekening mag maar uit 900 tekens bestaan.',
+	'invalid_date_of_birth' => 'Ongeldige geboortedatum ingevuld.',
+	'location_required' => 'Vul een locatie in.',
+	'location_minimum_2' => 'Je locatie moet minstens uit 2 tekens bestaan.',
+	'location_maximum_128' => 'Je locatie mag maar uit 120 tekens bestaan.',
```
Comment (Azoraqua) [inline styles/language/Dutch/language.php]: 120 should be 128.

File: styles/language/Dutch/language.php [hunk 3]
```diff
@@ -652,11 +655,11 @@
...
-    'new_reply' => 'Nieuw Reply',
-    'mod_actions' => 'Mod Actions',
-    'lock_thread' => 'Sluit Thread',
-    'unlock_thread' => 'Ontsluiten Thread',
-    'merge_thread' => 'Smelt Thread Samen',
+    'new_reply' => 'Nieuwe Antwoord',
+    'mod_actions' => 'Mod Acties',
+    'lock_thread' => 'Sluit Topic',
+    'unlock_thread' => 'Ontgrendel topic',
+    'merge_thread' => 'Smelt een topic Samen',
```
Comment (Azoraqua) [inline styles/language/Dutch/language.php]: "Samenvoegen" would be more accurate.

**Comment by jesseke55** [comment]
Thanks @Azoraqua for noticing this.

File: styles/language/Dutch/language.php [hunk 4]
```diff
@@ -254,18 +257,18 @@
     'query_errors_info' => 'De volgende fouten kunt u problemen diagnosticeren met uw interne server query.',
     'no_query_errors' => 'Geen query fouten gevonden',
     'date' => 'Datum:',
-    'port' => 'Port:',
-    'viewing_error' => 'Bekijk Error',
-    'confirm_error_deletion' => 'Bent u zeker dat u deze error wilt verwijderen?',
+    'port' => 'Poort:',
+    'viewing_error' => 'Bekijk melding',
+    'confirm_error_deletion' => 'Weet u hey zeker dat u deze error wilt verwijderen?',
```
Comment (Azoraqua) [inline styles/language/Dutch/language.php]: "Hey" should be "het".

Comment (jesseke55) [inline styles/language/Dutch/language.php]: Fixed.

**Comment by samerton** [comment]
Thanks for this

**Pull request merged**